data-raw/url_reference.csv

```diff
@@ -13,3 +13,5 @@ palmisano,2017-09-23,1,https://discovery.ucl.ac.uk/1575442/1/Palmisanoetal.zip
 radon,today,1,https://radon.ufg.uni-kiel.de/radondaily.txt
 radonb,today,1,https://radon-b.ufg.uni-kiel.de/radondaily.txt
 irdd,2018-08-13,1,https://zenodo.org/record/3367518/files/Chapple%2C%20RM%202019%20Catalogue%20of%20radiocarbon%20determinations%20%26%20dendrochronology%20dates%20%28August%202019%20Release%29.xlsx
+pacea,2020-01-22,1,http://www.paleoanthro.org/media/journal/content/PA20110001_S01.zip
+14c-palaeolithic2020-1-23,,1,https://ees.kuleuven.be/geography/projects/14c-palaeolithic/radiocarbon-palaeolithic-europe-database-v26-extract.xlsx
```

Comment:
Please use the same database name: `14c-palaeolithic` -> `14cpalaeolithic`

There also seems to be a table layout error in `14c-palaeolithic2020-1-23,,1`